CAFAna/Core/Binning.h

```diff
@@ -72,4 +72,28 @@ namespace ana
   const Binning kTrueEnergyBins = TrueEnergyBins();
   /// No entries for underflow or overflow
   const std::vector<double> kTrueEnergyBinCenters = TrueEnergyBinCenters();
+  
+  /// LBL and ND analyses binning
+  /// Eventually define as k{ND,FD}{Nue,Numu}Binning
+  const std::vector<double> binEEdges = {0.,  0.5,  1.,  1.25, 1.5, 1.75,
```

Comment:
Can all these have better names, and start with a k? In fact, your comment hints at that. Why not kNDNumuBinEdges?